maps/unused/oldmap.dmm

```diff
@@ -1605,7 +1605,7 @@
 "dI" = (
 /obj/table,
 /obj/item/device/radio/intercom{
-	
```

Comment:
???

Reply:
Im not sure how that one happened.

Reply:
Looks like old maps had a phantom space at a few places and vscode automatically removed them. Should be fine?